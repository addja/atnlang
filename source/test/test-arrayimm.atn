def main() {
	a = {1,2,34};
	print a, "%n";
	a = {"hola","yo","hey"};
	print a, "%n";
	a = {true, false, false};
	print a, "%n";
}
