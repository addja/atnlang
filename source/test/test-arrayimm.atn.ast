(PROGRAM (def main PARAMS (LIST_INSTR (:= a (ARRAY_DECL 1 2 34)) (print a "%n") (:= a (ARRAY_DECL "hola" "yo" "hey")) (print a "%n") (:= a (ARRAY_DECL true false false)) (print a "%n"))))