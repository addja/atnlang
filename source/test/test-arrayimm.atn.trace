main() <entry point>
return <line 7>
