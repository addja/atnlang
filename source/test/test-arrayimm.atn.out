[1,2,34]
[hola,yo,hey]
[true,false,false]
